app.js

```diff
@@ -0,0 +1,107 @@
+//Guessing Game, Three questions
+//Quetions one
+//Do you own an umbrella
+//If yes, You are new to the Seattle area
```

Comment:
This is ok for now, but in the future, remove all extraneous comments from code. These comments are good for notes to yourself when developing, but should not make it to GitHub.